lib/src/features/authentication/view/signup_view.dart

```diff
@@ -19,7 +20,16 @@ class SignUpView extends StatelessWidget {
         bottom: 30.0,
         left: 30.0,
         right: 30.0,
-        child: _buider(context),
+        child: BlocProvider<SignUpBloc>(
+          create: (context) {
+            return SignUpBloc();
+          },
+          child: BlocBuilder<SignUpBloc, SignUpState>(
```

Comment:
Just call `BlocBuilder` when you want to use `state`
Remember if use one property of state please use 'buildWhen'. If just use function please use 
```
buildWhen: (previous,current) => false,
```